### ISSUE TITLE
commons-logging should be added to pom.xml 

### ISSUE THREAD

**Issue by seanpd** [awslabs/amazon-kinesis-client#123]
This dependency should be called 

```
  <dependency>
      <groupId>commons-logging</groupId>
      <artifactId>commons-logging</artifactId>
      <version>1.1.1</version>
  </dependency>
```

I used the Maven Shade plugin to create a fat jar and found the commons-logging missing.

**Comment by pfifer** [comment]
Fix was merged in PR #124

**Comment by pfifer** [comment]
This is now fixed, and release in 1.7.3.

Feel free to reopen if you have any additional questions.

**Comment by danieljamesscott** [comment]
Is it possible to switch to slf4j? Or is this a major change?
